### PR TITLE
RXFactor TermsOfService

### PR DESCRIPTION
**Pull request by kornelijepetak**
## What's this?
RXFactors the TermsOfServiceViewModel

### Relationships
Deals with part of #3352 

## Why do we want this?
To make our lives easier.

## How is it done?
By, well... er... refactoring?

### Side effects
None expected.

## Review hints
I took special care to make this as simple as possible. What other hints do you need?

## :squid: Permissions
Please do.

### CODE REVIEW

File: Toggl.Giskard/Toggl.Giskard.csproj [hunk 1]
```diff
@@ -400,6 +400,9 @@
     <Compile Include="Activities\SignUpActivity.cs" />
     <Compile Include="TemplateSelectors\SelecCountryTemplateSelector.cs" />
     <Compile Include="Fragments\TermsOfServiceFragment.cs" />
+    <Compile Include="Fragments\TermsOfServiceFragment.ViewBinds.cs">
+	  <DependentUpon>TermsOfServiceFragment.cs</DependentUpon>
```
Comment (simonrozsival) [inline Toggl.Giskard/Toggl.Giskard.csproj]: Tabz?! 😱

File: Toggl.Giskard/Fragments/TermsOfServiceFragment.cs [hunk 2]
```diff
@@ -1,19 +1,25 @@
 ﻿using System;
+using System.Reactive.Disposables;
```
Comment (simonrozsival) [inline Toggl.Giskard/Fragments/TermsOfServiceFragment.cs]: 🔥

File: Toggl.Giskard/Fragments/TermsOfServiceFragment.cs [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
 using Toggl.Giskard.Extensions;
+using Toggl.Giskard.Extensions.Reactive;
 
 namespace Toggl.Giskard.Fragments
 {
     [MvxDialogFragmentPresentation(AddToBackStack = true)]
-    public sealed class TermsOfServiceFragment : MvxDialogFragment<TermsOfServiceViewModel>
+    public sealed partial class TermsOfServiceFragment : MvxDialogFragment<TermsOfServiceViewModel>, IReactiveBindingHolder
     {
+        public CompositeDisposable DisposeBag { get; } = new CompositeDisposable();
```
[the remark this answers is not in the document]
Comment (kornelijepetak) [reply Toggl.Giskard/Fragments/TermsOfServiceFragment.cs]: say what? What's wrong with this?

[the remark this answers is not in the document]
Comment (simonrozsival) [reply Toggl.Giskard/Fragments/TermsOfServiceFragment.cs]: OK 🤦‍♂️ this is Android, things work differently. Let's prettend this never happened.

**Pull request merged**